ROMFS/px4fmu_common/init.d/rc.interface

```diff
@@ -181,7 +177,8 @@ then
 	else
 		echo "ERROR  [init] Failed loading mixer: ${MIXER_FILE}"
 		echo "ERROR  [init] Failed loading mixer: ${MIXER_FILE}" >> $LOG_FILE
-		tone_alarm ${TUNE_ERR}
+		# Error tune.
+		tune_control play -t 2
```

Comment:
How are we supposed to know that `-t 2` is an error?

Reply:
I agree.  I think it would be nice if the tunes were an enum rather than an array.

Reply:
Perhaps default_tunes.cpp and tune_definition.h could be combined, and maybe to simplify things they could be condensed into tunes.cpp and tunes.h?

Reply:
Can we do that before adding more magic numbers everywhere? @simonegu please advise.

Reply:
@dagar, this looks like an appropriate time to work on the tone_alarm/tune_control refactor.  Updates to follow.

Reply:
@mcsauder 
> Perhaps default_tunes.cpp and tune_definition.h could be combined, and maybe to simplify things they could be condensed into tunes.cpp and tunes.h?

As far as I know those separate files exist such that manufacturers and individuals can easily create custom tunes without creating an unnecessary diff. It's one of those things "towards px4 modularity". Also I'm not sure what would be made simpler by merging all these files into `tunes.h`.